Api/User.py

```diff
@@ -21,12 +19,13 @@ def post(self):
             new_user = User(
                 data["name"],
                 data["username"],
+                data["phone_number"],
                 data["password"],
                 data["confirmpassword"]
             )
-            Exist = new_user.User_exist()
-            if Exist:
-                return Exist, 400
+            exist = new_user.user_exist()
```

Comment:
this line is redundant. To help reduce LOC try to figure out how to get rid of redundancy here.